overlays/builder.py

```diff
@@ -0,0 +1,21 @@
+from subprocess import call
+import os
+import shutil
+import glob
+
+def compile():
+    # The compile needs the include for the spi2 stuff, not enabling that at the moment
+    # as I don't want to pull in the include section from CHIP-dt-overlays
+    call(["dtc", "-O", "dtb", "-o", "overlays/chip-spi2.dtbo", "-b", "o", "-@", "overlays/chip-spi2.dts"])
```

Comment:
I'm pretty new to dt overlays -- what spi2 stuff is missing? For my edification, can you point me to it? Is it somewhere in [here](https://github.com/NextThingCo/CHIP-dt-overlays)?